### PR TITLE
docs: native speaker review

### PR DESCRIPTION
**Pull request by pasched**

### CODE REVIEW

File: doc/releases/v03.02.adoc [hunk 1]
```diff
@@ -4,35 +4,36 @@ https://www.youtube.com/watch?v=SJUhlRoBL8M[Think p(OSI)tive!]
...
+* Added a centralized documentation in https://github.com/OpenSimulationInterface/osi-documentation[OSI-Documentation].
+** Added contributors guidelines.
+** Added commenting style guidelines.
+** Added a uniform citation style based on APA.
+** Updated installation guides for osi in Linux and Windows.
+** Added a reference in sphinx based on generated doxygen documentation.
+* Added rules to the comments to enforce osi message validation with the https://github.com/OpenSimulationInterface/osi-validation[OSI-Validator].
+* Added the rule definitions in the rules.yml.
+* Modularized testing of commenting style into python unit tests.
+* Defined a standardized file format for reading trace files (*.osi).
```
Comment (stefancyliax) [inline doc/releases/v03.02.adoc:32]: @pasched So now we fix typos in the release notes? 😃

Comment (pasched) [reply doc/releases/v03.02.adoc:32]: @stefancyliax In the heat of the review battle I just fixed them all and did not think about the agreement to leave them as they were. Please ignore these changes. ;)

File: doc/releases/versioning.adoc [hunk 2]
```diff
@@ -1,23 +1,23 @@
-= Versioning and compatibility
+= Versioning and Compatibility
```
Comment (stefancyliax) [inline doc/releases/versioning.adoc]: @pasched Why was this rolled back? This aren't release notes of a older release.

**Review by stefancyliax — CHANGES_REQUESTED** [review]

File: doc/releases/versioning.adoc (hunk 2 above)
Comment (stefancyliax) [reply doc/releases/versioning.adoc]: Fixed it myself.

**Review by stefancyliax — APPROVED** [review]
Looks fine now. Going to merge.

**Pull request merged**